src/main/java/com/diffblue/deeptestutils/DeeptestUtilsException.java

```diff
@@ -0,0 +1,34 @@
+package com.diffblue.deeptestutils;
+
+// Copyright 2016-2018 Diffblue limited. All rights reserved.
+
+/**
+ * <code>DeeptestUtilsException</code> is used to wrap other exceptions
+ * thrown from Deeptestutils that are not directly required for
+ * the test.
+ *
+ * @author <a href="http://diffblue.com">Diffblue</a>
+ */
+public class DeeptestUtilsException extends RuntimeException {
```

Comment:
Should this inherit from `RuntimeException`? 

*Reasons for extending RuntimeException*
+ these shouldn't ever happen, them happening is an error test-gen, they are not expected exceptions
+ we don't have to have the test declare that this is a possible thing to throw which keeps the test cleaner

*Reasons for extending Exception*
- if you are using this library and you are not test-gen, these are wrappers around exceptions you might want to catch and so should probably be part of the interface. 
- we can never change from this without breaking backwards compatibility (old tests won't say `throws DeeptestUtilsException`), we can change to this. 

I think on balance this is the right choice, but the last one gave me pause for thought so worth at least considering this carefully. @peterschrammel do you have an opinion?